packages/atomic/src/components/facets-v1/atomic-numeric-facet/atomic-numeric-facet.tsx

```diff
@@ -179,6 +182,16 @@ export class AtomicNumericFacet
     this.formatter = event.detail;
   }
 
+  @Listen('atomic/numberInputApply')
```

Comment:
I know I'll get this comment again 😄  so: The decorator options cannot be variables, that's why the even name are hardcoded with primitives

Reply:
Also, I'm using an event because passing a method as a prop to a component gives a build warning